src/renderer/prefComponents/imageUploader/index.vue

```diff
@@ -152,8 +180,21 @@ export default {
   & .el-input {
     max-width: 242px;
   }
+  & .el-button.btn-reset,
   & .button-group {
     margin-top: 30px;
   }
+  & .pref-cb-legal-notices {
+    &.smms {
+      margin-bottom: 30px;
+    }
+    &.github {
+      margin-top: 30px;
+    }
+    &.error {
+      border: 1px solid var(--themeColor);
```

Comment:
How about use `outline` instead of `border`, because border will change the height of current element.